src/types/License.ts

```diff
@@ -0,0 +1,8 @@
+import { LicenseStatus } from './LicenseStatus';
+
+export interface License {
+    id: number;
+    display_name: string;
+    status: LicenseStatus;
+    created_at: string;
+}
```

Comment:
@oleg-semyonov I wanted to expose as less as possible info about the license for now. Will this be enough data for you?